modules/python3/bindings/src/pyvolume.cpp

```diff
@@ -144,7 +144,101 @@ void exposeVolume(pybind11::module& m) {
         .def_property_readonly(
             "data", static_cast<const py::array& (VolumePy::*)() const>(&VolumePy::data));
 
-    py::bind_vector<VolumeSequence>(m, "VolumeSequence", py::module_local(false));
+    py::class_<VolumeSequence>(m, "VolumeSequence", py::module_local(false))
+        .def(py::init<>())
+        .def(py::init<std::span<std::shared_ptr<const Volume>>>(), py::arg("volumes"))
+        .def(py::init<const VolumeSequence&>(), "Copy constructor")
+
+        .def("__getitem__",
+             [](const VolumeSequence& v, std::ptrdiff_t i) -> std::shared_ptr<const Volume> {
+                 if (i < 0 && (i += v.size()) < 0) {
```

Comment:
Why the special handling of negative indices? Does this cover the cases like `v[-1]` ?

Use parenthesis around right condition as i is modified?

Reply:
Yes, this is for [-1] in python